.codeclimate.yml

```diff
@@ -15,7 +13,13 @@ ratings:
   paths:
   - "src/**/*.js"
 exclude_paths:
-- dist/**/*
-- node_modules/**/*
-- test/**/*
-- coverage/**/*
+- '.github/'
+- 'dist/'
+- 'test/'
```

Comment:
This change isn't removing scores for tests on codeclimate, as they were never scored before (Lines 13-14). This is just removing them from showing up on codeclimate without any score.

As far as running codeclimate on the tests, I think it would just add a whole host of problems that we can't do anything about because tests have a very repetitive structure.